services/addons/tests/firmware_manager/test_commsignia_upgrader.py

```diff
@@ -77,6 +77,61 @@ def test_commsignia_upgrader_upgrade_success(mock_sshclient, mock_scpclient):
     # Assert notified success value
     notify.assert_called_with(success=True)
 
+@patch("addons.images.firmware_manager.commsignia_upgrader.SCPClient")
+@patch("addons.images.firmware_manager.commsignia_upgrader.SSHClient")
+@patch("addons.images.firmware_manager.commsignia_upgrader.time")
+def test_commsignia_upgrader_upgrade_success_post_update(mock_time, mock_sshclient, mock_scpclient):

```

Comment:
You may want to add a test for the scenario that someone doesn't include an "ALL OK" in the script to verify the code handles that properly.

Reply:
Added a unit test for failed post_upgrade.sh run